package.json

```diff
@@ -1,11 +1,56 @@
 {
   "name": "ilo_base_theme",
   "description": "Scaffolding tools for ILO Base Theme.",
+  "version": "1.0.0",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/international-labour-organization/ilo_base_theme"
+  },
+  "contributors": [
+    {
+      "name": "Antonio De Marco",
+      "email": "antonio@nuvole.org",
+      "url": "https://www.nuvole.org"
+    },
+    {
+      "name": "Massimo Altafini",
+      "email": "massimo@nuvole.org",
+      "url": "https://www.drupal.org/u/mxt"
+    }
+  ],
   "scripts": {
-    "postinstall": "patch-package && make install-design-system"
+    "postinstall": "patch-package && make install-design-system",
+    "theme:watch": "webpack --config=webpack.config.dev.js --watch",
+    "theme:build": "webpack --config=webpack.config.dev.js",
+    "theme:dist": "webpack --config=webpack.config.prod.js --mode=production"
   },
   "dependencies": {
     "@ilo-org/twig": "1.2.4",
     "patch-package": "^8.0.0"
-  }
+  },
+  "devDependencies": {
+    "@babel/core": "^7.25.2",
+    "@babel/preset-env": "^7.25.4",
+    "autoprefixer": "^10.4.20",
+    "babel-loader": "^9.2.1",
+    "css-loader": "^7.1.2",
+    "css-minimizer-webpack-plugin": "^7.0.0",
+    "file-loader": "^6.2.0",
+    "glob": "^11.0.0",
+    "mini-css-extract-plugin": "^2.9.1",
+    "postcss": "^8.4.47",
+    "postcss-assets-webpack-plugin": "^4.1.2",
+    "postcss-loader": "^8.1.1",
+    "postcss-sort-media-queries": "^5.2.0",
+    "sass": "^1.79.3",
+    "sass-loader": "^16.0.2",
+    "terser-webpack-plugin": "^5.3.10",
+    "url-loader": "^4.1.1",
+    "webpack": "^5.94.0",
+    "webpack-cli": "^5.1.4",
+    "webpack-remove-empty-scripts": "^1.0.4"
+  },
+  "browserslist": [
+    "last 2 versions"
+  ]
```

Comment:
I think we need an [engines](https://docs.npmjs.com/cli/v10/configuring-npm/package-json#engines) key to specify the node version we are supporting and also to make package module type  to utilise `ESM` instead of commonjs

Reply:
Fixed